src/antsibull/collection_meta.py

```diff
@@ -8,8 +8,12 @@
 """
```

Comment:
You missed some here

``` diff
diff --git a/src/antsibull/tagging.py b/src/antsibull/tagging.py
index 4f50611..2a58905 100644
--- a/src/antsibull/tagging.py
+++ b/src/antsibull/tagging.py
@@ -13,7 +13,7 @@ import asyncio
 import os
 import re
 import sys
-import typing as t
+from collections.abc import AsyncGenerator
 
 import asyncio_pool  # type: ignore[import]
 
@@ -25,8 +25,8 @@ from antsibull_core.yaml import store_yaml_file, load_yaml_file
 
 from antsibull.collection_meta import CollectionMetadata, CollectionsMetadata
 
-TAG_MATCHER: t.Pattern[str] = re.compile(r'^.*refs/tags/(.*)$')
-TAG_VERSION_REGEX: t.Pattern[str] = re.compile(r'^v?(.*)$')
+TAG_MATCHER: re.Pattern[str] = re.compile(r'^.*refs/tags/(.*)$')
+TAG_VERSION_REGEX: re.Pattern[str] = re.compile(r'^v?(.*)$')
 mlog = log.fields(mod=__name__)
 
 
@@ -142,7 +142,7 @@ async def _get_collection_tags(
     if not repository:
         flog.debug("'repository' is None. Exitting...")
         return data
-    tag_version_regex: t.Pattern[str] | None = None
+    tag_version_regex: re.Pattern[str] | None = None
     if meta_data.tag_version_regex:
         try:
             tag_version_regex = re.compile(meta_data.tag_version_regex)
@@ -158,7 +158,7 @@ async def _get_collection_tags(
     return data
 
 
-async def _get_tags(repository) -> t.AsyncGenerator[str, None]:
+async def _get_tags(repository) -> AsyncGenerator[str, None]:
     flog = mlog.fields(func='_get_tags')
     args = (
         'git',
@@ -196,7 +196,7 @@ async def _get_tags(repository) -> t.AsyncGenerator[str, None]:
 
 
 def _normalize_tag(
-    tag: str, regex: t.Pattern[str] | None
+    tag: str, regex: re.Pattern[str] | None
 ) -> str | None:
     regex = regex or TAG_VERSION_REGEX
     if match := regex.match(tag):
```

```
curl https://paste.sr.ht/blob/9b46ceb83eb8351795e1668b7285c6e1b2aa613f | git apply
```

Reply:
Thanks! Implemented in 66fa3fae519bc5a491c7297f8320d6ebb7cb6966.